src/Web3/Types.elm

```diff
@@ -218,3 +218,32 @@ type CallType
     | Async
     | Setter
     | Getter
+
+
+{-| Available ethereum denominations.
+-}
+type EthDenomination
+    = Wei --        1
+    | Kwei --       1000
```

Comment:
Oh, I see, so just shortening the list. That list is exactly the list specified by Ethereum [themselves](https://github.com/ethereum/web3.js/blob/0.15.0/lib/utils/utils.js#L40), so I think they are exactly the set we should use. So I don't think we should remove any of them.